apis/r/tools/r-ci.sh

```diff
@@ -372,15 +372,24 @@ DumpLogsByExtension() {
     fi
     extension=$1
     shift
-    package=$(find . -maxdepth 1 -name "*.Rcheck" -type d)
-    if [[ ${#package[@]} -ne 1 ]]; then
-        echo "Could not find package Rcheck directory, skipping log dump."
+
+    # Note: this script may be run from the apis/r subdirectory or from the
+    # package base directory. This runs in CI from a clean image so in the
+    # (hypothetical) case of multiple log files, we err on the side of providing
+    # the operator with more inforation rather than less.
```

Comment:
There is a way to echo with a colon and some other marker to create the separate little sections in a scrollable log. I thought I used that somewhere but I can't locate it right now, and can't find it in the docs for GHA. 😿

Reply:
Sounds like a nice to have!

Reply:
Happy to take this mod on a follow-on PR

Reply:
This seems to be it 
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines

Reply:
Thanks!

https://github.com/single-cell-data/TileDB-SOMA/pull/2453/commits/1b22704a67041de98bacfbb0ca75f63e59925600